pkg/objstore/swift/swift.go

```diff
@@ -251,7 +251,7 @@ func configFromEnv() SwiftConfig {
 		ProjectName:       os.Getenv("OS_PROJECT_NAME"),
 		UserDomainID:      os.Getenv("OS_USER_DOMAIN_ID"),
 		UserDomainName:    os.Getenv("OS_USER_DOMAIN_NAME"),
-		ProjectDomainID:   os.Getenv("OS_PROJET_DOMAIN_ID"),
```

Comment:
Interesting... Let's maybe add that to CHANGELOG?

BTW Are you using SWIFT? (:

Reply:
Ups, I'll move it to other PR or at least to the https://github.com/thanos-io/thanos/pull/2665
Thanks for pointing out

Reply:
Yes, I didn't want to originally but the only available larger storage in our company fitting our needs is Swift currently and we are running out of space in MinIO which we use right now. 

Not sure how many people uses Thanos with Swift. Hopefully it will work :crossed_fingers: :)

Reply:
removed the change